docs/src/modules/components/DiamondSponsors.js

```diff
@@ -91,6 +91,7 @@ export default function DiamondSponsors() {
                 display: 'flex',
                 alignItems: 'center',
                 justifyContent: 'center',
+                borderRadius: '12px',
```

Comment:
It feels a bit weird with the dividers
![image](https://github.com/mui/material-ui/assets/45398769/96b21dee-c899-4764-a20b-ebb40c31d525)

What about adding a `margin-right`/`margin-left` of `12px` on the dividers?

![image](https://github.com/mui/material-ui/assets/45398769/210dfc86-487c-4236-9194-28105ee1b56e)